.github/workflows/pull_requests.yml

```diff
@@ -37,9 +41,21 @@ jobs:
         run: dotnet restore
       - name: Build
         run: dotnet build Letterbook.ActivityPub.sln
-#        TODO: test results and coverage
       - name: Test
-        run: dotnet test Letterbook.ActivityPub.sln
+        run: dotnet test Letterbook.ActivityPub.sln --collect:"XPlat Code Coverage" --results-directory "coverage"
```

Comment:
I think the recommendation is to do this kind of thing with 
`if: succeed() || fail()`, so that we still get results when tests fail.

Reply:
Let me know if I'm not understanding, but I'm using `if always()` to save test artifacts. Is there another behavior you think should always be occurring?